src/gate_detect/test/gate-detect-test.cpp

```diff
@@ -0,0 +1,27 @@
+//
+// Created by Cameron on 03/04/18.
+//
+
+#include <gate_detect/gateDetectMsg.h>
+#include <gtest/gtest.h>
+
+TEST(gate_detect, addMessage) {
+    gate_detect::gateDetectMsg msg;
+
+    msg.detectLeft    = 1;
+    msg.angleLeft     = 2.0f;
+    msg.distanceLeft  = 2.0f;
+    msg.detectRight   = 1;
+    msg.angleRight    = 2.0f;
+    msg.distanceRight = 2.0f;
+    msg.detectTop     = 1;
+    msg.angleTop      = 2.0f;
+    msg.distanceTop   = 2.0f;
+
+    EXPECT_TRUE(true);
```

Comment:
Unless something explicitly fails (ie. like a `EXPECT_TRUE` or `ASSERT_TRUE`) this test will pass. The actual "test" here is really just that this file compiles, so we should add a note at the top to that effect and remove this line.

Reply:
removed expect_true, and added comments